src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -599,8 +599,26 @@ private IConfidentialClientApplication BuildConfidentialClientApplication(Merged
 
             try
             {
+                IList<string> AddContinuousAccessEvaluationCapability(IEnumerable<string> clientCapabilities)
+                {
+                    if (clientCapabilities == null)
+                    {
+                        return new[] { Constants.CaeCapability };
+                    }
+                    else
+                    {
+                        var finalClientCapabilities = new List<string>(clientCapabilities);
```

Comment:
I am not convinced that this logic is well unit tested. Granted that the unit tests here are hard to read, but do you have test for both branches?

Reply:
Added additional test cases.